pages/index.tsx

```diff
@@ -1,5 +1,8 @@
+import Navbar from "../components/commons/Navbar/Navbar"
 export default function Home() {
   return (
-    <div>Squeryll</div>
+    <div>
+      <Navbar/>
+    </div>
```

Comment:
```suggestion
    <>
      <Navbar/>
    </>
```

That `div` was just adding an unnecessary element in the DOM. Use Fragments instead for such cases.